docs/project-config.md

```diff
@@ -1,51 +1,42 @@
 # Project Configuration
 
-A Relay will generally not accept or forward events where it doesn't know the project. See [_Getting Started_](index.md) for a short introduction to project configs. This page enumerates all possible options.
+A Relay will generally not accept or forward events where it doesn't know the project. See [_Getting Started_](index.md) for a short introduction to project configs.
```

Comment:
> A Relay will generally not accept or forward events where it doesn't know the project.

I think this is no longer true and depends on the mode. Can you remove this sentence and we follow up with proper documentation of the relay modes and config?

Reply:
`index.md` seems to explain this pretty well already, this might as well just become API documentation